EidosScribe/EidosTextView.mm

```diff
@@ -2001,16 +2001,22 @@ - (NSMutableArray *)completionsForKeyPathEndingInTokenIndex:(int)lastDotTokenInd
 	
 	// First, a sorted list of globals
 	for (auto symbol_sig : *terminus->Properties())
-		[candidates addObject:[NSString stringWithUTF8String:symbol_sig->property_name_.c_str()]];
+	{
+		if (!symbol_sig->deprecated_)
```

Comment:
These diffs should not be coming through in your PR.  I think there is a problem with your rebasing.  These are changes that I committed.  I'm going to stop here until this is resolved.

Reply:
It is because those commits are absent from the `windows_port` branch, which is what I am PRing into. You will have to merge `master` into `windows_port` (or just delete the branch and then regenerate it so it is even with `master`. Or just leave it and let the commits be added from this PR.. This might lead to a bit more of a confusing commit history though, I'm not sure. I'm also not exactly a master of git..

Reply:
Ah!  Right you are.  I have just merged master's changes into windows_port.  Let me know when things are sorted out; let's do the separate PR of the gnulib files first, as discussed below.  Thanks!

Reply:
I'm not sure why these are still here. Time to make a backup and get into the deep git.

Reply:
Actually, as a test I tried just opening a new PR and looks like this issue is gone. So it is just this PR which is a bit confused. So I'm just going to abandon this PR, and make a new one with `gnulib` (as discussed), and then another with everything else. I will keep this PR open for now to keep our conversation here open.

Reply:
OK, SGTM.  I've seen this problem before with rebasing pull requests; it seems to either not work well, or be easy to do incorrectly.  @petrelharp may have advice.  Anyway, I will refrain from doing any further commits until we have merged in your gnulib PR, and then life ought to be somewhat simpler.